DCHdigi/README.md

```diff
@@ -0,0 +1,20 @@
+# Drift chamber (DCH) digitizers
+
+## DCHsimpleDigitizer
+
+* Similar to DCHdigi_v01, but it uses the placement matrix of the wires to apply the smearing along/perpendicular to the sense wire
+* No cluster counting information added into the digitized output
+* It relies on a dedicated data extension, similar to DCHdigi_v01
+
+## DCHdigi description
+
```

Comment:
Can you add that "this digitizer is meant to be used with `DriftChamber_o1_v02` and is expected to work for the upcoming `DriftChamber_o1_v03`"